src/restconfig/src/main/java/org/geoserver/rest/catalog/NamespaceController.java

```diff
@@ -152,6 +133,17 @@ public void namespacePut(
     
             new CatalogBuilder(catalog).updateNamespace(nsi, namespace);
             catalog.save(nsi);
+
+            // make sure workspace isolation value is updated too
+            WorkspaceInfo workspace = catalog.getWorkspaceByName(namespace.getPrefix());
```

Comment:
In the catalog there is already a NamespaceWorkspaceConsistencyListener, wondering if code like this may be better put there?